src/app/(proper_react)/(redesign)/MobileShell.module.scss

```diff
@@ -10,6 +10,8 @@
   display: none;
 
   @media screen and (max-width: tokens.$screen-xl) {
+    box-shadow: tokens.$box-shadow-xs;
+    border-bottom: 1px solid tokens.$color-grey-20;
```

Comment:
I'm having trouble seeing the box shadow, but the bottom border is also visible without the redesign flag.

Reply:
See https://github.com/mozilla/blurts-server/pull/5671#discussion_r1981288584.